docs/releases/1.8.10.txt

```diff
@@ -9,4 +9,4 @@ Django 1.8.10 fixes several bugs in 1.8.9.
 Bugfixes
 ========
 
-* ...
+* Checked query name clashes of hidden relationships (:ticket:`26162`).
```

Comment:
To make it more obvious where the checking happens, I'd say, "Added system checks for query name clashes..."